src/mainwindow.cpp

```diff
@@ -100,6 +112,85 @@ void MainWindow::updateBoms(DbManager *d)
     }
 }
 
+bool MainWindow::refillMatchFromSaveFileToDatabase(const std::string& path)
+{
+    // Read and stock all clubs
+    std::ifstream matchFlow(path.c_str(), std::ios::in);
+    if (!matchFlow)
+    {
+        std::cerr << "Failed to open Matchs.txt for reading!" << std::endl;
+        return false;
+    }
+
+    int firstScore, secondScore;
+    std::string firstTeam, secondTeam;
+    int ind1(1), ind2(1);
+
+    std::string line;
+    while (getline(matchFlow, line, '\n'))
+    {
+        std::istringstream iss(line);
+        std::string aString;
+        int count(-1);
+        while (getline(iss, aString, '/'))
+        {
+            count++;
+            if (count == 0)
+                continue;
+
+            if (count == 1)
+                continue;
+
+            switch (count % 4)
+            {
+                case 2:
+                    firstTeam   =  aString; break;
+                case 3:
+                    firstScore  = atoi(aString.c_str()); break;
+                case 0:
+                    secondScore = atoi(aString.c_str()); break;
+                case 1:
+                    secondTeam  =  aString; break;
+            }
+
+            if ((count % 4) == 1)
+            {
+                unsigned int firstTeamInd, secondTeamInd;
+                for (unsigned int i = 0; i < aTeamsId.size(); ++i)
```

Comment:
the aTeamsId size should be called once and stored in a const before the loop

Reply:
long unsigned int teamSize = aTeamsId.size();
for (unsigned int i = 0; i < teamSize; ++i)

To avoid warning, like Displays branch